tests/test_ptc.py

```diff
@@ -414,6 +414,15 @@ def notest_covAstier(self):
                 self.assertEqual(
                     ptc.covariancesModelNoB[ampName].shape, covModelNoBShape
                 )
+            # Check if evalPtcModel produces expected values
+            nanMask = ~np.isnan(ptc.finalMeans[ampName])
+            means = ptc.finalMeans[ampName][nanMask]
+            covModel = ptc.covariancesModel[ampName][nanMask]
+            covariancesModel = ptc.evalPtcModel(means, setBtoZero=False)[ampName]
+            self.assertFloatsAlmostEqual(covariancesModel, covModel, atol=1e-13)
+            # Note that the PhotoTransferCurveDataset does not store the gain
+            # fit parameter for FULLCOVARIANCE with b=0, so we can't compare
+            # exactly.
```

Comment:
Is that something that we need to add?

Reply:
Yes we do, but I want to keep this ticket small and then filed this ticket to update the ptc class https://rubinobs.atlassian.net/browse/DM-46509